### ISSUE TITLE
Alert boxes require some accessibility fixes

### ISSUE THREAD

**Issue by slowbot** [bloom-housing/bloom#2249]
**Describe the solution you'd like**
Alert boxes should be enhanced to meet the needs of screen reader users

**What is the acceptance criteria/definition of done?**
- SVGs for alert boxes can be marked with area-hidden
- Alert box close action should be labeled "dismiss error" or "close alert"

**QA Review Instructions**
1. Enable VoiceOver on Mac
2. Launch a short form application
3. Intentionally trigger an error
4. Screen reader should not identify the icon and should apply focus to the close icon which should read "dismiss error" or "close alert"

**Comment by slowbot** [comment]
@jaredcwhite LGTM
